frontend/public/components/utils/tile-view-page.jsx

```diff
@@ -227,6 +235,17 @@ const filterByGroup = (items, filters) => {
   );
 };
 
+export const filterByGroupByDropdown = (items, groupBy) => {
+  if (groupBy === groupByTypes.Operator) {
+    const installedOperators = _.filter(items, (item) => item.kind === 'InstalledOperator');
+    const nonOperators = _.filter(items, (item) => item.kind !== 'InstalledOperator');
+    const groupedOperators = _.groupBy(installedOperators, (item) => item.obj.csv.metadata.name);
```

Comment:
Maybe we can sanitize the operator name somehow to make it look better on the UI. The title currently looks really odd.

Reply:
@rohitkrai03 Do you think the version is making it look odd? cc @serenamarie125

Reply:
@rohitkrai03  Let me know if this looks okay 
![Screenshot from 2019-12-19 23-40-18](https://user-images.githubusercontent.com/22490998/71198055-fb14e980-22b8-11ea-9f46-82fe9250fb4f.png)

Reply:
Yeah, this looks much better.

Reply:
Done